packages/api/amplify_api/example/integration_test/util.dart

```diff
@@ -184,24 +201,7 @@ Future<Post> addPostAndBlog(
 ) async {
   const name = 'Integration Test Blog with a post - create';
   final createdBlog = await addBlog(name);
-
-  final post = Post(title: title, rating: rating, blog: createdBlog);
-  final createPostReq = ModelMutations.create(
-    post,
-    authorizationMode: APIAuthorizationType.userPools,
-  );
-  final createPostRes =
-      await Amplify.API.mutate(request: createPostReq).response;
-  expect(createPostRes, hasNoGraphQLErrors);
-  final data = createPostRes.data;
-  if (data == null) {
-    throw Exception(
-      'Null response while creating post. Response errors: ${createPostRes.errors}',
-    );
-  }
-  postCache.add(data);
-
-  return data;
+  return addPost(title, rating, createdBlog);
```

Comment:
request: Could you make a similar change here as well in `addPostAndBlog`? https://github.com/aws-amplify/amplify-flutter/blob/next/packages/api/amplify_api/example/integration_test/util.dart#L188

Reply:
nm, my mistake